service/k8sclient/k8client.go

```diff
@@ -295,6 +296,23 @@ func (c *K8Client) DeleteXtraDBCluster(ctx context.Context, name string) error {
 	return c.kubeCtl.Delete(ctx, res)
 }
 
+// RestartXtraDBCluster restarts Percona XtraDB cluster with provided name.
+func (c *K8Client) RestartXtraDBCluster(ctx context.Context, name string) error {
```

Comment:
Can we write test(not only API) to these methods to ensure that cluster is really restarted?

Reply:
Added.